edith-step/src/UserChart.test.js

```diff
@@ -1,42 +1,68 @@
+import {enableFetchMocks} from 'jest-fetch-mock'
+enableFetchMocks();
+import {mount} from 'enzyme';
 import React from 'react';
-import { mount, unmount } from 'enzyme';
 import ReceiptInput from './ReceiptInput';
```

Comment:
should we delete this import ? Seems like it has not been used.

Reply:
Yea, I deleted it.